File: app/Flecha Digital/Model/Viaje.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Flecha_Digital.Model
{
    public class Viaje
    {
        public int no_servicio { get; set; }
        public int cve_linea { get; set; }
        public string nombre_linea { get; set; }
        public int origen { get; set; }
        public int destino { get; set; }
        public string descripcion_clase { get; set; }
        public DateTime fecha_salida { get; set; }
        public DateTime fecha_llegada { get; set; }
        public string descripcion_servicios { get; set; }
        public string descripcion_viaje { get; set; }
        public double costo_total { get; set; }
    }
}

File: server/services/viajes.api/models/classes/viaje.js
class Viaje{
    constructor(no_servicio, origen_viaje, destino_viaje, tipo_viaje, fecha_salida, duracion, fecha_llegada){
        this.no_servicio = no_servicio;
        this.origen_viaje = origen_viaje;
        this.destino_viaje = destino_viaje;
        this.tipo_viaje = tipo_viaje;
        this.fecha_salida = fecha_salida;
        this.duracion = duracion;
        this.fecha_llegada = fecha_llegada;
    }
    constructor(viaje){
        this.no_servicio = viaje.no_servicio;
        this.origen_viaje = viaje.origen_viaje;
        this.destino_viaje = viaje.destino_viaje;
        this.tipo_viaje = viaje.tipo_viaje;
        this.fecha_salida = viaje.fecha_salida;
        this.duracion = viaje.duracion;
        this.fecha_llegada = viaje.fecha_llegada;
    }
}


module.exports = Viaje;